commands/lastsave.md

```diff
@@ -1,7 +1,7 @@
 Return the UNIX TIME of the last DB save executed with success.
 A client may check if a `BGSAVE` command succeeded reading the `LASTSAVE` value,
 then issuing a `BGSAVE` command and checking at regular intervals every N
-seconds if `LASTSAVE` changed.
+seconds if `LASTSAVE` changed. Redis considers the database saved successfully at startup.
```

Comment:
``` suggestion
seconds if `LASTSAVE` changed. Redis considers the database saved successfully at startup. `LASTSAVE` returns the UNIX TIME of the server start.
```

Reply:
look goods, i am not very good at english, how about something like this?

Redis considers the database saved successfully at startup and `LASTSAVE` will return the UNIX TIME of the server start by default.

Reply:
Updated!

Reply:
thanks for picking this up, LGTM